nixos/modules/services/misc/bazarr.nix

```diff
@@ -0,0 +1,80 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bazarr;
+in
+{
+  options = {
+    services.bazarr = {
+      enable = mkEnableOption "bazarr";
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/bazarr/data";
+        description = "The directory where bazarr stores its data files.";
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the bazarr web interface.";
+      };
+
+      listenPort = mkOption {
+        type = types.port;
+        default = 6767;
+        description = "Port on which the bazarr web interface should listen";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "bazarr";
+        description = "User account under which bazarr runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "bazarr";
+        description = "Group under which bazarr runs.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' 0700 ${cfg.user} ${cfg.group} - -"
+    ];
+
+    systemd.services.bazarr = {
+      description = "bazarr";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        Type = "simple";
+        User = cfg.user;
+        Group = cfg.group;
+        ExecStart = pkgs.writeShellScript "start-bazarr" ''
```

Comment:
I think this will create log entries in the journal that get prefixed with `<NIX_STORE_HASH>-start-bazaarr`. May I suggest using `pkgs.writeShellScriptBin "bazarr" [...]` (and tacking on "/bin/bazarr" at the end) to make the journal more pleasing to read?

Reply:
Or set `serviceConfig.SyslogIdentifier = "bazarr"`. Not sure what is preferred.